packages/shared/src/components/Feed.spec.tsx

```diff
@@ -232,23 +231,13 @@ const renderComponent = (
       >
         <LazyModalElement />
         <SettingsContext.Provider value={settingsContext}>
-          <OnboardingContext.Provider
-            value={{
-              myFeedMode: OnboardingMode.Manual,
-              isOnboardingOpen: false,
-              onCloseOnboardingModal: jest.fn(),
-              onInitializeOnboarding: jest.fn(),
-              onShouldUpdateFilters: jest.fn(),
-            }}
-          >
-            <Toast autoDismissNotifications={false} />
-            <Feed
-              feedQueryKey={['feed']}
-              feedName={feedName}
-              query={ANONYMOUS_FEED_QUERY}
-              variables={variables}
-            />
-          </OnboardingContext.Provider>
+          <Toast autoDismissNotifications={false} />
+          <Feed
+            feedQueryKey={['feed']}
```

Comment:
We have OtherFeed or some constant for this?

Reply:
The key you mean?
Probably didn't really change anything here 😅

Reply:
non blocking just checking if we have constant in the meantime 👍

Reply:
I actually checked, but we don't specific that one somewhere at the moment as enum.